html-intro/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Intro</title>
+  </head>
+  <body>
+    <img
+      src="https://static.pokemonpets.com/images/monsters-images-300-300/2133-Shiny-Eevee.webp"
+      alt="Shiny Eevee" />
+    <h1>Shiny Eevee</h1>
+    <p>© Pokémon. ® Nintendo.</p>
```

Comment:
These symbols should be using HTML entities rather than pasting them in directly